packages/angular/projects/ui-angular/src/lib/primitives/button/button.component.ts

```diff
@@ -28,6 +29,10 @@ export class ButtonComponent implements OnInit {
     if (this.fullWidth) {
       className += ` amplify-button--fullwidth`;
     }
+    if (this.isDisabled) {
+      className += ` amplify-button--disabled amplify-button--loading`;
+    }
```

Comment:
Doesn't have to be this PR, but I think it'd make sense to create `isLoading` `@Input` as well when we build primitives. Authenticator doesn't use it, but others might.